### PR TITLE
Integration test for spk deployment get

### LINKED ISSUE #891: spk deployment create and get integration test
**As a:**
Developer

**I want:**
To validate `spk deployment create` and `spk deployment get` in the integration tests

**So that:**
I am confident about the functionality

**Describe the solution you'd like:**
Add the integration tests [here](https://github.com/CatalystCode/spk/tree/master/tests).
- [x] Update the tasks in the pipelines to include the step that runs `spk deployment create`
- [x] test the data retrieved from `spk deployment get` to make sure data was inserted and retrieved

**Acceptance Criteria:**
The test run successfully in the existing test pipeline.

**Does this require updates to documentation?:**
Yes

### PR DESCRIPTION
**Pull request by samiyaakhtar**
**Note**: a sample smoke test against this branch is [here](https://dev.azure.com/epicstuff/hellobedrock/_build/results?buildId=11422&view=logs&j=8fe65ec1-57b5-590c-8e73-b2e70046a92a&t=5273a394-895a-588d-3853-b9fce385fa20&l=271). 
- Removed some logging statements that were logging storage key to console
- Removed duplicated code from `introspection-validations.sh` and merged it with `validations.sh`
- The common code for introspection and services is in `validations.sh` and unrelated code in `introspection-validations.sh`, such as onboard command smoke test

Closes https://github.com/microsoft/bedrock/issues/891

### CODE REVIEW

**Comment by andrebriggs** [comment]
@samiyaakhtar looks good. Have you tried running this with the latest release of `spk`?

**Comment by samiyaakhtar** [comment]
@andrebriggs I have, and I ran into a [classic missing null check](http://dev.azure.com/epicstuff/hellobedrock/_build/results?buildId=11268&view=logs&j=8fe65ec1-57b5-590c-8e73-b2e70046a92a&t=5273a394-895a-588d-3853-b9fce385fa20&l=266) unfortunately, I've opened a PR for it [here](https://github.com/microsoft/spektate/pull/65), and will need to consume the new package in spk (sorry, may need another release) 😬 
Once that is done I'm hopeful this should work

**Comment by andrebriggs** [comment]
@samiyaakhtar so we need another release before merging this?

**Comment by samiyaakhtar** [comment]
@andrebriggs Yes, we would, but no rush this task can wait till we make the next one if it's not anytime soon.

**Comment by samiyaakhtar** [comment]
Here's a [sample smoke test](https://dev.azure.com/epicstuff/hellobedrock/_build/results?buildId=11422&view=logs&j=8fe65ec1-57b5-590c-8e73-b2e70046a92a&t=5273a394-895a-588d-3853-b9fce385fa20&l=271) against this PR. Looking for feedback and review on this one
cc @mtarng @edaena @andrebriggs

File: src/commands/deployment/get.ts [hunk 1]
```diff
@@ -387,7 +387,7 @@ export const printDeployments = (
     });
 
     // tslint:disable-next-line: no-console
-    console.log("\n" + table.toString());
+    console.log(table.toString());
```
Comment (dennisseah) [inline src/commands/deployment/get.ts:390]: thanks for fixing this.

**Review by edaena — APPROVED** [review]

**Review by NathanielRose — APPROVED** [review]

**Pull request merged**